pallets/pallet-bonded-coins/src/traits.rs

```diff
@@ -0,0 +1,20 @@
+use frame_support::{dispatch::DispatchResult, traits::fungibles::Inspect};
+
+/// Copy from the Polkadot SDK. once we are at version 1.13.0, we can remove this.
+pub trait ResetTeam<AccountId>: Inspect<AccountId> {
```

Comment:
Why does the trait itself need to require `Inspect<AccountId>`?

Reply:
Don't know. Doesn't really make sense. I just copied it over from the polkadot SDK